drivers/axi_core/axi_dac_core/axi_dac_core.h

```diff
@@ -52,12 +53,14 @@ struct axi_dac {
 	uint32_t base;
 	uint8_t	num_channels;
 	uint64_t clock_hz;
+	struct axi_dac_channel *channels;
```

Comment:
You can comment the structures to understand how to use the `channels` parameter.

Reply:
Commenting the entire structure means creating separate commit/PR, since this is responsible only of adding a new member to the struct. I will comment here only the added member to the struct.